### ISSUE TITLE
OSM Schema not doing anything

### ISSUE THREAD

**Issue by JRegimbal** [Shared-Reality-Lab/IMAGE-server#481]
The `Header_Info` key conveys the min and max of the map region's latitude and longitude, however it is *not* included in the schemas and encodes these numerical values as text. That is, the min and max latitude would be conveyed as "From xxxxx To yyyyy", which requires string manipulation to extract.

The schema should be updated to properly state and, when necessary, require information from a response and should encode these bounds in a clearer way. For example:
```
{
  "bounds": {
    "latitude": {
      "min": xxxx,
      "max": yyyy
    }
  ...
  }
}
```

**UPDATE**: Taking another look at the schema file, it doesn't do anything. The "street_info" root key is not required and does not exist in the returned data, meaning that nothing is actually being validated in practice. This must also be corrected.

**Comment by jeffbl** [comment]
@samueljohnsegun148 If I move this to Oct03 sprint is that workable? Or will this need more time?

**Comment by samueljohnsegun148** [comment]
@jeffbl That should be okay as this is being addressed in #490, except that it can't be merged yet because of the changes (possibility of introducing the "distance" key to the request schema) being proposed for the OSM preprocessor, which we are still working on.

**Comment by jeffbl** [comment]
@samueljohnsegun148 I believe we decided that the distance key will not go in now since we can't reasonably populate it anyway. If my recollection is correct, does that mean this can move forward?

**Comment by samueljohnsegun148** [comment]
Yes @jeffbl. I think it can.

**Comment by JRegimbal** [comment]
Seems to be closed in #490.
